optimum/intel/openvino/modeling_diffusion.py

```diff
@@ -889,9 +889,7 @@ def reshape(
             )
 
         if self.text_encoder_3 is not None:
-            self.text_encoder_3.model = self._reshape_text_encoder(
-                self.text_encoder_3.model, batch_size, getattr(self.tokenizer_3, "model_max_length", -1)
-            )
+            self.text_encoder_3.model = self._reshape_text_encoder(self.text_encoder_3.model, batch_size, -1)
```

Comment:
why not `getattr(self.tokenizer_3, "model_max_length", -1)`

Reply:
in our test model model_max_length is 256, while default argument in pipeline_call__ is 512